src/components/FormField/FormField.js

```diff
@@ -1,49 +1,59 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { useField } from 'formik';
 import styles from './FormField.module.scss';
-import { isNullable } from '~/utils/check-value';
+import mainStyles from '~/assets/styles/main';
+import FormFieldErrorBoundary from '../error-boundaries/FormFieldErrorBoundary';
 
 const FormField = ({
   label,
-  type,
   isEditable,
-  onChange,
-  value,
+  name,
+  type,
   placeholder,
 }) => {
+  const [field, meta] = useField(name);
+
   const inputProps = {
     readOnly: !isEditable,
-    onChange: isEditable ? onChange : null,
+    label,
+    name,
+    type,
```

Comment:
You don't need `type` for custom input props, cause `textarea` don't have `type` property

Reply:
Fixed, thanks